Dockerfile

```diff
@@ -1,42 +1,45 @@
 FROM amazon/aws-lambda-python:3.7
 
+SHELL ["/bin/bash", "-euxo", "pipefail", "-c"]
+
 # Set up working directories
-USER root
-RUN mkdir /app
-RUN chown -R root /app
-RUN mkdir -p /app/build
-RUN mkdir -p /app/bin
+RUN mkdir /app /app/build /app/bin
 
+# Install dependencies and download libraries we need to run in lambda
+WORKDIR /tmp
+RUN yum install -y --setopt skip_missing_names_on_install=False \
+        wget \
+        yum-utils \
+        zip \
+    && \
+    yumdownloader --setopt skip_missing_names_on_install=False -x \*i686 --archlist=x86_64 \
+        clamav \
+        clamav-lib \
+        clamav-update \
+        gnutls \
+        json-c \
+        libtasn1 \
+    && \
+    yum clean all && \
+    rpm2cpio clamav-0*.rpm | cpio -idmv && \
+    rpm2cpio clamav-lib*.rpm | cpio -idmv && \
+    rpm2cpio clamav-update*.rpm | cpio -idmv && \
+    rpm2cpio gnutls*.rpm | cpio -idmv && \
+    rpm2cpio json-c*.rpm | cpio -idmv && \
+    rpm2cpio libtasn1*.rpm | cpio -idmv && \
+    rm -fv *.rpm && \
+    # Copy over the binaries and libraries
+    mv -v /tmp/usr/bin/clamscan /tmp/usr/bin/freshclam /tmp/usr/lib64/* /app/bin/ && \
+    # Fix the freshclam.conf settings
+    echo "DatabaseMirror database.clamav.net" > /app/bin/freshclam.conf && \
+    echo "CompressLocalDatabase yes" >> /app/bin/freshclam.conf && \
+    :
```

Comment:
Prepending, rather than appending, the line with `&&` makes this trailing no-op unnecessary and is, in my opinion, slightly easier to read and update, [cf](https://github.com/nginxinc/docker-nginx/blob/5ce65c3efd395ee2d82d32670f233140e92dba99/mainline/debian/Dockerfile).

Alternatively, since this is run under `set -e`, the `&&` could be replaced with `;` to reduce the visual noise a bit, [cf](https://github.com/docker-library/postgres/blob/ef45b990868d5a0053bd30fdbae36551b46b76c9/15/bullseye/Dockerfile).

Reply:
Yes, all true - just how I'm used to formatting these things. Maybe I'll change it one day.

Also worth noting that Docker will now accept heredoc format for `RUN` commands, which is especially nice when running with `set -e`.

Reply:
(The latter requires a special Dockerfile syntax pragma.)